kvdb/etcd_test.go

```diff
@@ -137,6 +139,10 @@ func TestEtcd(t *testing.T) {
 	for _, test := range tests {
 		test := test
 
+		if test.debugOnly && !etcdDebug {
```

Comment:
Isn't this test now always running with `etcdDebug true`, making this check unnecessary?

Reply:
Discussed offline - test can still be running without the `dev` tag outside of CI.